pandas/core/strings.py

```diff
@@ -2151,12 +2151,54 @@ def encode(self, encoding, errors="strict"):
         return self._wrap_result(result)
 
     _shared_docs['str_strip'] = ("""
-    Strip whitespace (including newlines) from each string in the
-    Series/Index from %(side)s. Equivalent to :meth:`str.%(method)s`.
+    Remove leading and trailing characters.
+
+    Strip whitespaces (including newlines) or a set of specified characters
+    from each string in the Series/Index from %(side)s.
+    Equivalent to :meth:`str.%(method)s`.
+
+    Parameters
+    ----------
+    to_strip : str, optional
+        Specifying the set of characters to be removed.
+        All combinations of this set of characters will be stripped.
+        Default value is None, which means whaitspaces will be removed.
 
     Returns
     -------
     stripped : Series/Index of objects
+
+    See Also
+    --------
+    str.slice : Slice substrings from each element in the Series/Index
```

Comment:
`slice` doesn't seem very related to strip.

Not sure if there is anything else related, but we should have here `Series.str.strip`, `Series.str.lstrip` and `Series.str.rstrip`, so they link each other.